extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -915,7 +918,7 @@ export class SchemaCompareMainWindow {
 			}
 
 			this.updateSourceAndTarget();
-			this.deploymentOptions = result.deploymentOptions;
```

Comment:
was this causing issue too? that seems similar in effect..

Reply:
no but I thought it would be best to use this function wherever the deployment options get set